src/main/kotlin/me/nalab/api/core/TimeUtil.kt

```diff
@@ -0,0 +1,34 @@
+package me.nalab.api.core
+
+import java.time.Clock
+import java.time.Instant
+import java.time.ZoneId
+import java.time.format.DateTimeFormatter
+
+object TimeUtil {
+
+    private var clock: Clock? = null
+
+    fun toInstant(): Instant {
+        var current = Instant.now()
+        if (clock != null) {
+            current = Instant.now(clock)
+        }
+        return formatTo6Digit(current)
+    }
+
+    private fun formatTo6Digit(instant: Instant): Instant {
+        val formatter = DateTimeFormatter.ofPattern("yyyy-MM-dd'T'HH:mm:ss.SSSSSSX")
+            .withZone(ZoneId.of("UTC"))
+        return Instant.parse(formatter.format(instant))
+    }
+
+    fun fixed(clock: Clock?) {
```

Comment:
fixed 할 때는 not null type으로 받기
```suggestion
    fun fixed(clock: Clock) {
```

Reply:
[bcc6961](https://github.com/depromeet/na-lab-server/pull/395/commits/bcc696122032b778fab1f939f835fb656a70b38a)